EcommerceApp/backend/spring-boot-ecommerce-app/src/main/java/kkramarenko/ecommerceapp/enums/DiscountRate.java

```diff
@@ -0,0 +1,9 @@
+package kkramarenko.ecommerceapp.enums;
+
+public enum DiscountRate {
+    starter,
+    bronze,
+    silver,
+    gold,
+    platinum;
```

Comment:
enums should be written in capital letters

Reply:
Done